packages/esm-patient-attachments-app/src/utils.ts

```diff
@@ -30,7 +30,8 @@ export function createGalleryEntry(data: AttachmentResponse): Attachment {
   return {
     id: data.uuid,
     src: `${window.openmrsBase}${attachmentUrl}/${data.uuid}/bytes`,
-    title: data.comment,
+    title: data.comment?.[0] == '{' ? JSON.parse(data.comment)['name'] : data.comment,
```

Comment:
This would cause errors if JSON.parse fails, for example if someone mistakenly labels their image "{Left Foot". The safer thing would be to wrap the JSON.parse in a try catch block. Also we should allow a fallback to the file name if data.comment['name'] is undefined.

Reply:
Removed the JSON parsing all along.